File: order-service/src/main/java/com/intexsoft/webshop/orderservice/service/StatusService.java
package com.intexsoft.webshop.orderservice.service;

import com.intexsoft.webshop.orderservice.dto.status.StatusDto;
import com.intexsoft.webshop.orderservice.model.enums.StatusName;

public interface StatusService {
    StatusDto addStatusToOrderWithId(Long orderId, StatusName statusName);
}

File: order-service-kt/src/main/kotlin/com/intexsoft/webshop/orderservicekt/service/StatusService.kt
package com.intexsoft.webshop.orderservicekt.service

import com.intexsoft.webshop.orderservicekt.dto.status.StatusDto
import com.intexsoft.webshop.orderservicekt.model.enums.StatusName

interface StatusService {
    fun addStatusToOrderWithId(orderId: Long, statusName: StatusName): StatusDto
}
